fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseFactory.java

```diff
@@ -67,8 +67,13 @@
 
     @Nullable
     private Executor networkExecutor = null;
+
+    @Nonnull
     private Executor executor = ForkJoinPool.commonPool();
 
+    @Nonnull
+    private Function<Executor, Executor> contextExecutor = exec -> exec;
```

Comment:
```suggestion
    private Function<Executor, Executor> contextExecutor = Function.identity();
```

Reply:
(I believe IntelliJ would offer the same suggestion.)

Reply:
It does, but I was on the fence about whether or not I preferred the more explicit syntax (seeing how `Function.identity()` simply returns `x -> x`).   Anyway, updated.